constrain/demo/library_item_demo/library_verification_cases.json

```diff
@@ -204,12 +204,12 @@
             "datapoints_source": {
                 "idf_output_variables": {
                     "T_oa_db": "Environment:Site Outdoor Air Drybulb Temperature [C](TimeStep)",
-                    "T_chw": "CoolSys1 Supply Outlet Node:System Node Temperature [C](TimeStep)",
-                    "m_chw": "CoolSys1 Supply Outlet Node:System Node Mass Flow Rate [kg/s](TimeStep)"
+                    "temperature_water_chilled": "CoolSys1 Supply Outlet Node:System Node Temperature [C](TimeStep)",
+                    "flow_mass_water_chilled": "CoolSys1 Supply Outlet Node:System Node Mass Flow Rate [kg/s](TimeStep)"
                 },
                 "parameters": {
-                    "T_oa_max": 21.11,
-                    "T_oa_min": 12.78,
+                    "temperature_air_outdoor_max": 21.11,
+                    "temperature_air_outdoor_min": 12.78,
                     "T_chw_max_st": 8.89,
```

Comment:
temperature_water_chilled_max_setpoint

Reply:
We've decided
1. clean up the```library_verification_cases.json```
2. Should include ```CHReset``` verification item which is currently missing.